Cargo.toml

```diff
@@ -136,6 +136,7 @@ turbopack-tests = { path = "crates/turbopack-tests" }
 turbopack-wasm = { path = "crates/turbopack-wasm" }
 turbopath = { path = "crates/turborepo-paths" }
 turborepo = { path = "crates/turborepo" }
+turborepo-analytics = { path = "crates/turborepo-analytics" }
```

Comment:
We needed a new package because turborepo-cache and turborepo-lib needed to use it.